security-response.md

```diff
@@ -0,0 +1,107 @@
+# Security Response Guidelines
+
+Security vulnerabilities should be handled quickly and sometimes privately. The
+primary goal of this process is to reduce the total time users are vulnerable to
+publicly known exploits.
+
+The OpenTelemetry (OTel) Technical Committee (TC) and relevant repo maintainers,
+supported by tooling provided by the Sig-Security, are responsible for
+responding to the incident organizing the entire response including internal
+communication and external disclosure.
+
+## Supported Versions
+
+The OTel project provides community support only for the last minor
+version: bug fixes are released either as part of the next minor version or as
+an on-demand patch version. Independent of which version is next, all patch
+versions are cumulative, meaning that they represent the state of our `main`
+branch at the moment of the release. For instance, if the latest version is
+0.10.0, bug fixes are released either as part of 0.11.0 or 0.10.1.
+
+Security fixes are given priority and might be enough to cause a new version to
+be released.
+
+## Disclosures
+
+### Private Disclosure Processes
+
+In order for the vulnerability reports to reach maintainers as soon as possible,
+the preferred way is to use the `Report a vulnerability` button on the
+`Security` tab in the respective GitHub repository. It creates a private
+communication channel between the reporter and the maintainers.
+
+If you are absolutely unable to or have strong reasons not to use GitHub
+reporting workflow, please reach out to the Technical Committee using
+[cncf-opentelemetry-tc@lists.cncf.io](mailto:cncf-opentelemetry-tc@lists.cncf.io)
+and we will provide instruction on how to report the vulnerability using an
+encrypted message.
+
+[gh-organization]: https://github.com/open-telemetry
+
+### Public Disclosure Processes
+
+If you know of a publicly disclosed security vulnerability please IMMEDIATELY
+email
+[cncf-opentelemetry-tc@lists.cncf.io](mailto:cncf-opentelemetry-tc@lists.cncf.io)
+to inform the Security Response Committee (SRC) about the vulnerability so they
+may start the patch, release, and communication process.
+
+If possible the TC will ask the person making the public report if the issue can
+be handled via a private disclosure process. If the reporter denies the request,
+the TC will move swiftly with the fix and release process. In extreme cases you
+can ask GitHub to delete the issue but this generally isn't necessary and is
+unlikely to make a public disclosure less damaging.
+
+## Patch, Release, and Public Communication
+
+For each vulnerability a member of the TC will volunteer to lead coordination
+with the relevant repo owners, and is responsible for sending disclosure
+emails to the rest of the community. This TC member will be referred to as the Fix Lead.
```

Comment:
Are emails the best channel here or would we rather put this on Slack and/or the website? I wonder how many OTel users are really subscribed to the mailing lists on lists.cncf.io.